env/env.js

```diff
@@ -77,6 +77,7 @@ const development = removeNullValues({
   TWILIO_DIVERT: 1,
   REACT_APP_MOCK: 1,
   ALLOY_MOCK: 1,
+  MG_DOMAIN: DEV.MG_SANDBOX_DOMAIN,
```

Comment:
If you just want something for local dev, I would probably just add it to `env.dev.sample.js`.  Also, I would add in the `MG_API_KEY`.  The secrets have the same fields between DEV, STAGING, and PRODUDCTION, and I want to keep that, so unless we plan to have separate values for all of them, let's stick to keeping dev overrides in `env.dev.sample.js`